Admin/AdminInterface.php

```diff
@@ -1024,4 +1024,19 @@ public function checkAccess($action, $object = null);
      *
      */
     // public function configureActionButtons($action, $object = null);
+
+    /**
+     * Hook to handle access authorization.
+     *
+     * @param string $action
+     * @param object $object
+     */
+    public function hasAccess($action, $object = null);
```

Comment:
Please comment your methods in this interface, so we can merge this without a BC break.